runtime/laos/src/lib.rs

```diff
@@ -122,7 +110,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	authoring_version: 1,
 	spec_version: 1201,
 	impl_version: 0,
-	apis: RUNTIME_API_VERSIONS,
+	apis: apis::PUBLIC_RUNTIME_API_VERSIONS,
```

Comment:
it might sound like too much, but do you think it is worth adding a test for  the `apis::PUBLIC_RUNTIME_API_VERSIONS` value? so we ensure it doesn't change. If not why?

Reply:
yes, not nice: it's forced by the fact that the macro defines `RUNTIME_API_VERSIONS` not public. I am trying to expose the problem to the polkadot-sdk guys. We'll see if they accept it.
We can choose another name for sure...  :D